ng-add fails if it can't find tslint

### Describe the problem

When performaing ```ng add @fortawesome/angular-fontawesome@0.6.0``` it asked for what font icons I would like:

```
Installing packages for tooling via npm.
Installed packages for tooling via npm.
? Choose Font Awesome icon packages you would like to use: Free Solid Icons, Free Regular Icons, Free Brands Icons
UPDATE package.json (3013 bytes)
UPDATE src/app/app.module.ts (11139 bytes)
```

But afterwards failed with: ```Cannot find module 'tslint'```

### What did you expect?

It to succeed without any errors. I do not use tslint.

### Reproducible test case

Unable to produce a test case on stackblitz, requires command line.

--- a/projects/schematics/src/ng-add/index.ts
+++ b/projects/schematics/src/ng-add/index.ts
@@ -66,7 +66,12 @@ function addModule(projectName?: string): Rule {
     });
     host.commitUpdate(recorder);
 
-    context.addTask(new TslintFixTask(modulePath, {}));
+    /* tslint is required to add a tslint fix task */
+    try {
+      require('tslint');
+      context.addTask(new TslintFixTask(modulePath, {}));
+    } catch (err) {
+    }
 
     return host;
   };

--- a/projects/schematics/src/ng-add/index.ts
+++ b/projects/schematics/src/ng-add/index.ts
@@ -71,6 +71,7 @@ function addModule(projectName?: string): Rule {
       require('tslint');
       context.addTask(new TslintFixTask(modulePath, {}));
     } catch (err) {
+      context.logger.warn('Formatting was skipped because tslint is not installed.');
     }
 
     return host;

